templates/vite/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "template-vite",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "xprebuild": "fuels build",
+    "dev": "vite",
+    "build": "pnpm xprebuild && tsc -b && vite build",
+    "lint": "eslint .",
+    "preview": "vite preview",
```

Comment:
Do we need `preview`? JW so we have parity with the `nextjs` template commands.

Reply:
Hmm, I think we can remove it probably.